README.md

```diff
@@ -319,52 +319,26 @@ page](https://github.com/xournalpp/xournalpp/releases).
+The file extension `.xopp` is a gzipped XML file. PDFs are not embedded into the file, so if the PDF is deleted, the background is lost. `.xopp` is basically the same file format as `.xoj`, which is used by Xournal. Therefore, Xournal++ is able to read `.xoj` files, and can also export to `.xoj`. As soon as notes are exported to a `.xoj` file, all Xournal++ specific extensions like additional background types, are lost.
 
-\*.xopp can theoretically be read by Xournal, as long as you do not use any new feature. Xournal does not open files that contain new attributes or unknown values, so Xournal++ will add the extension .xopp to all saved files to indicate the potential presence of Xournal++-only features.
+`.xopp` files can theoretically be read by Xournal, as long as you do not use any new features. Xournal does not open files that contain new attributes or unknown values, so Xournal++ will add the extension `.xopp` to all saved files to indicate the potential presence of Xournal++-only features.
 
-All new files will be saved as _.xopp. If an _.xoj file that was created by Xournal is opened, the Save-As dialog will be displayed on save. If the \*.xoj file was created by Xournal++, the file will be overwritten on save and the file extension will not change.
+All new files will be saved as `.xopp`. If an `.xoj` file that was created by Xournal is opened, the Save-As dialog will be displayed on save. If the `.xoj` file was created by Xournal++, the file will be overwritten on save and the file extension will not change.
 
 **We are currently introducing a new file format that can efficiently store attached PDF files and other attachments internally. We will still allow for attachments that are linked to external files. Please refer to [#937](https://github.com/xournalpp/xournalpp/issues/937) for further details.**
 
-## Development
-
-For developing new features, create an issue or comment on an existing issue to let others know what you are doing.
-For development, create a fork and use the master as base. Create a pull request for each fix.
-Do not create big pull requests, as long as you don't break anything features also can be
-merged, even if they are not 100% finished.
-
-See [GitHub:xournalpp](http://github.com/xournalpp/xournalpp) for current development. You can also join
-our Gitter channel via the badge on top.
-
-Also take a look at our [coding conventions](https://github.com/xournalpp/xournalpp/wiki/Coding-conventions)
-
-## Code documentation
+## Building
 
-The code documentation is generated using Doxygen.
+We support building on five operating systems:
 
-In order to generate the documentation yourself, first install Doxygen and graphviz, i.e.
+- [Linux](readme/LinuxBuild.md)
+- [MacOS](readme/MacBuild.md)
+- [Windows](readme/WindowsBuild.md)
+- [Android](https://gitlab.com/TheOneWithTheBraid/xournalpp_mobile#getting-started)
+- [iOS](https://gitlab.com/TheOneWithTheBraid/xournalpp_mobile#getting-started)
```

Comment:
Xournal++ Mobile is not the same project as Xournal++, so it should not be part of the build steps.